includes/Core/Util/Reset.php

```diff
@@ -139,11 +139,23 @@ private function delete_all_plugin_options() {
 	 */
 	private function delete_all_user_metas() {
 		global $wpdb;
+
 		$user_query = new \WP_User_Query(
 			array(
-				'fields'   => 'id',
-				'meta_key' => $wpdb->prefix . OAuth_Client::OPTION_ACCESS_TOKEN,
-				'compare'  => 'EXISTS',
+				'fields'     => 'id',
+				'meta_query' => array(
+					'relation' => 'OR',
+					// Keys are un-prefixed in network mode.
+					array(
+						'key'     => OAuth_Client::OPTION_ACCESS_TOKEN,
+						'compare' => 'EXISTS',
+					),
+					// Keys are prefixed in single site and multisite when not in network mode.
+					array(
+						'key'     => $wpdb->get_blog_prefix() . OAuth_Client::OPTION_ACCESS_TOKEN,
+						'compare' => 'EXISTS',
+					),
+				),
```

Comment:
I think it'd be more appropriate to just clear one or the other variant, depending on a `Context` check whether network mode is active or not.

Reply:
Sounds good, I've pushed an update here to do this instead.